infra/src/test/kotlin/fi/fta/geoviite/infra/MetaDataIT.kt

```diff
@@ -28,67 +27,66 @@ class MetaDataIT: DBTestBase() {
     }
 
     @Test
-    fun versioningWorks() {
-        assertEquals(null, getName(TEST_ROLE_CODE))
+    fun `Versioning works`() {
```

Comment:
Tämä testi tosiaan vaati vähän muutoksia. Muutin sen nyt käyttämään `user_group`-kenttää nimen sijasta, eli käyttötapaus muuttui todennäköisesti epärealistisemmaksi.

Reply:
Joo tämähän ei sinällään yritäkään testata usereita vaan tietokannan versiointia. Useria vain käytetään koska se on pieni rivi joten helppo luoda. Tämän voisi jopa tehdä niin että loisi täällä testissä ihan oman testitaulun ja lisäisi siihen versioinnin. Laiskuuttani kirjoittanut joskus näin...